### ISSUE TITLE
cannot edit text on any kind of signs

### ISSUE THREAD

**Issue by HaloSense** [AllOfFabric/All-of-Fabric-3#11]
I tried to categorise my storage today and when I placed an oak sign, it directly appeared on the ground/wall and the text editing GUI didn't pop up. This was later confirmed for all kinds of signs in another save.

**Comment by Ben64** [comment]
Shift right click

**Comment by HaloSense** [comment]
> Shift right click

nope, it doesn't work. the sign still appears as a blank one.
btw u might be getting me wrong. i'm not trying to put it on a container, this happens even when i try to put a sign on solid walls.

**Comment by Ben64** [comment]
Shift right click on a placed sign to edit it.

**Comment by HaloSense** [comment]
> Shift right click on a placed sign to edit it.

thanks it works.
still i do wonder why it is like this. is it related to some mods?

**Comment by AK9Official** [comment]
Most likely removing that mod in next update as it confuses people.

**Comment by HaloSense** [comment]
> Most likely removing that mod in next update as it confuses people.

No, keeping it can likely benefit most people, but some in-game reminders would help a lot.

**Comment by HaloSense** [comment]
> Most likely removing that mod in next update as it confuses people.

At least I don't need to remove the signs first to change the text XD
